test/common.yml

```diff
@@ -1,9 +1,13 @@
 name: test
 domain: 127.0.0.1.nip.io
 ldap:
+  disabled: true
   adminGroup: NA1
 kubernetes:
-  version: v1.15.3
+  version: v1.16.3
```

Comment:
we want to keep this as v1.15.3 or maybe bump it to v1.15.7/8, test-next will override thise and test on v1.16.3